types/Config.js

```diff
@@ -109,13 +109,14 @@ export type InitialOptions = {|
   setupFiles?: Array<Path>,
   setupTestFrameworkScriptFile?: Path,
   silent?: boolean,
-  skipNodeResolution: boolean,
+  skipNodeResolution?: boolean,
   snapshotSerializers?: Array<Path>,
   testEnvironment?: string,
   testFailureExitCode?: string | number,
   testMatch?: Array<Glob>,
   testNamePattern?: string,
   testPathIgnorePatterns?: Array<string>,
+  testPathDirs?: Array<Path>,
```

Comment:
i'm not sure if it's deprecated or missing.
After properly typing this object, flow found that we use this property in the code but it was not present in this type